robot.py

```diff
@@ -20,6 +21,24 @@
 from components.hang import Hang
 
 
+def scale_value(
+    value: float,
+    input_range: Tuple[float, float],
```

Comment:
Is there a particular reason why these are passed in as tuples? (Especially since the tuples would need to be constructed at some point, and then you need to access the individual elements...)